src/js/espn-borischentiers.js

```diff
@@ -31,15 +31,47 @@ window.addEventListener('load', function () {
   let promises = Object.keys(links).map(pos => setTierInfo(pos));
   Promise.all(promises)
     .then(_responses => {
-      addTierHeader();
-      const rows = document.getElementsByClassName(TABLE_BODY_CLASS)[0].rows;
-      for (let row of rows) {
-        addTierTd(row);
-      }
+      widenTable();
+      addBorisChenTiersHTML();
+      addRefreshButton();
     })
     .catch(_e => { });
 });
 
+function widenTable() {
+  const tableWrapper = document.getElementsByClassName(CONTAINER_CLASS)[0];
+  tableWrapper.style.width = '105%';
+}
+
+function addRefreshButton() {
+  const playersTableControls = document.getElementsByClassName(PLAYERS_TABLE_CONTROLS_CLASS)[0];
+  let newButton = document.createElement('button');
+  newButton.innerHTML = 'Refresh';
+  newButton.className = 'btn';
+  newButton.style = 'margin-left: 10px;';
+  newButton.onclick = function () {
+    removeOldTiersHTML();
+    addBorisChenTiersHTML();
+  };
+  playersTableControls.appendChild(newButton);
+}
+
+function removeOldTiersHTML() {
+  const rows = document.getElementsByClassName(TABLE_BODY_CLASS)[0].rows;
+  removeTierHeader();
```

Comment:
This is super nitpicky, so feel free to ignore it, but can we make `removeTierHeader();` the first statement in the function to match how the code looks in the `addBorisChenTiersHTML()` function?